erpnext_com/www/docs/user/manual/en/erpnext_integration/google_contacts.md

```diff
@@ -0,0 +1,21 @@
+<!-- add-breadcrumbs -->
+# Google Contacts Integration
+
+ERPNext provides an integration with Google Contacts in order for all users to synchronize their Google Contacts with ERPNext.
+
+
+## How to set up Google Contacts Integration
+
+In order to allow a synchronization with Google Contacts, you need to authorize ERPNext to get Contacts data from Google. Google Contacts Integration is set up with the following steps:
+
+1. Create a new project on Google Cloud Platform and generate new OAuth 2.0 credentials.
+<img class="screenshot" src="/docs/assets/img/erpnext_integrations/google_contacts_project_creation.gif">
```

Comment:
indent the images with tabs, else bullet breaks

Reply:
Don't want a bullet point for images, images are aligned with its description